docs/DeveloperGuide.md

```diff
@@ -487,6 +487,19 @@ testers are expected to do more *exploratory* testing.
     1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
        Expected: Similar to previous.
 
+3. Deleting persons while all persons are being shown
+
+    1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
```

Comment:
I maybe mistaken, but I thought the second point about the filtered list refers to that case. I thought it would be too cumbersome to add more because this section talks about sample manual tests and the list doesn't have to be exhaustive. The reader can interpret that since find works like that for single index the same would apply for multiindex deletion.

Reply:
That makes sense to me, thanks! Feel free to ignore this then :)